src/main/java/io/sentry/demos/example/Application.java

```diff
@@ -70,15 +71,15 @@ public void CheckoutCart(@RequestHeader(name = "X-Session-ID", required = true)
 
 		MDC.put("sessionId", sessionId);
 		MDC.put("transactionId", transactionId);
-		logger.info("Called checkout");
```

Comment:
this was getting logged as an individual Event, but we're already logging the error (and soon a transaction).

Use `System.out.print(...)` for server logging.